src/vaibhav/MinMaxNumber.java

```diff
@@ -0,0 +1,75 @@
+package vaibhav;
+
+public class MinMaxNumber {
+	
+	int minNumber(int[] arr) {		
+		int min=arr[0];
+		for (int index = 0 ;index < arr.length; index++){			
+			if(arr[index] < min)
+				min=arr[index];
+			else continue;			
+		}		
+		return min;		
+	}
+	
+	int maxNumber(int[] arr) {
+		int max=arr[0];
+		for (int index = 0 ;index < arr.length; index++){			
+			if(arr[index] > max)
+				max=arr[index];
+			else continue;			
+		}		
+		return max;	
+		
+	}
+	
+	int avgAllNumber(int [] arr) {
+		int sum=0;
+		int avg=0;
+		for (int index = 0 ;index < arr.length; index++ ){
+			sum+=arr[index];			
+		}
+		avg= sum / arr.length;
+		return avg;		
+	}
+	
+	int avgMinMaxNumber(int min,int max) {
+		int ans=0;		
```

Comment:
I think we have requirement where you need to find max & min number then you need to find avg of both, user will give us array, and you need to find min & max value. and then avg of both, I hope you got my point.

int avgOfMinMax = (getMinNumber(input)+getMaxNumber(input))/2